tests/unit/OrgPerson.spec.ts

```diff
@@ -108,8 +106,7 @@ const validOrgData = {
     lastName: '',
     middleName: '',
     organizationName: 'Test Org',
-    partyType: 'organization',
-    taxId: '1111'
```

Comment:
At the moment, Incorporation/Registration Number and Business Number are not displayed for incorporators.